cmd/training-operator.v1/main.go

```diff
@@ -187,3 +193,17 @@ func setupControllers(mgr ctrl.Manager, enabledSchemes controllerv1.EnabledSchem
 		}
 	}
 }
+
+func ValidateCRD(mgr ctrl.Manager, gvk schema.GroupVersionKind) {
```

Comment:
```suggestion
func validateCRD(mgr ctrl.Manager, gvk schema.GroupVersionKind) {
```

If it isn't needed, no exporting this function would be better.